src/editor.cpp

```diff
@@ -0,0 +1,580 @@
+#include "editor.hpp"
+
+#include "exceptions.hpp"
+#include "logger.hpp"
+#include "utils.hpp"
+
+#include "vpdecc/vpdecc.h"
+
+namespace vpd
+{
+using namespace types;
+using namespace constants;
```

Comment:
Using namespace here, because using it very frequently

Reply:
Still use it in place.